File: fixtures/scrapy_output/post_same_field_multiple_times.py
from scrapy import Request, Spider


class MySpider(Spider):
    name = 'myspider'

    def start_requests(self):
        body = [
            ('foo', 'bar'),
            ('foo', ''),
            ('foo', 'barbar'),
        ]
        body = '&'.join(key + '=' + value for key, value in body)
        yield Request(
            'http://example.com/',
            callback=self.parse,
            method='POST',
            body=body,
        )

    def parse(self, response):
        pass

File: fixtures/elixir_output/post_same_field_multiple_times.ex
request = %HTTPoison.Request{
  method: :post,
  url: "http://example.com/",
  options: [],
  headers: [],
  params: [],
  body: [
    {~s|foo|, ~s|bar|},
    {~s|foo|, ""},
    {~s|foo|, ~s|barbar|}
  ]
}

response = HTTPoison.request(request)
